### DISCUSSION TITLE
Federated default bundle size

### DISCUSSION THREAD

**Discussion by jfdesgagne** [callstack/repack#806] [Q&A] [answered]
Hello! Quick question: are empty federated bundles expected to be ~140kb (minimized) of webpack boilerplate code ? Any way of reducing it? Considering repack for federated micro-cx, but the additional 140kb for small visual bundles make it hard to justify. Thanks!

**Comment by jbroma** [comment, marked as answer]
Hi @jfdesgagne, this is the current limitation of MF, but with [hoisted MF runtime](https://module-federation.io/configure/experiments.html#hoisted-runtime) this should be greatly reduced (since runtime will be capable of being shared across the remotes)
